crates/frontend/src/components/override_form.rs

```diff
@@ -117,7 +117,7 @@ where
                                                             .map(ConfigType::DefaultConfig)
                                                             .collect::<Vec<_>>(),
                                                     )
-                                                    .expect("can't parse default config key");
+                                                    .unwrap_or(json!(input_value));
```

Comment:
@Datron we would also need to make the similar changes at line number 35 and 51 in `crates/frontend/src/components/context_form/utils.rs `

Reply:
@sauraww why? There is no type templates in context form

Reply:
If this is an issue, it seems out of scope of this PR